### ISSUE TITLE
Question: HTTP external Access

### ISSUE THREAD

**Issue by Dletta** [plasma-umass/browsix#68]
I have tried to understand whether I could run an HTTP server in my browser that others in my network (or even publicly) could access. I am unable to find this in clear terms in your documentation.

Will I be able to serve a static website from my browser to others in the web?

**Comment by ids1024** [comment]
This would be a nice feature. But I'm almost certain browsers don't provide an API that would allow this. (Unless it was communicating with a piece of software running outside the browser that could forward traffic in some way.)

**Comment by Dletta** [comment]
@ids1024 Yes, that makes sense. The socket API seemed to imply you could make an HTTP connection. It was unclear whether that was about inside a website to a local http client or not.

**Comment by ids1024** [comment]
Although I haven't tried it, I believe the socket API in Browsix is only usable for IPC. You can make a request to a server running inside Browsix from another program inside Browsix.

**Comment by Dletta** [comment]
That makes sense. You think the team will answer here? Otherwise will close the issue

**Comment by bitnom** [comment]
We were talking about this over in the gundb gitter. In theory, an adapter could be made to work with webrtc. Will I be the one to make that happen? Probably not. Would it be awesome? Probably yes.
